backend/graminsta/post/services.py

```diff
@@ -0,0 +1,37 @@
+"""
+Service functions for post module
+"""
+
+import datetime
+from django.contrib.auth import get_user_model
+from .models import Post
+
+
+def create_post(publisher_id, description, img):
+    """Create or update a Post in datebase.
```

Comment:
The documentation doesn't match the method name? `create or update` vs `create`. And do separate `create_post` and `update_post`.

Reply:
Changed to create